### PR TITLE
PMM-9938 Add cases for tooltips on settings page

### PR DESCRIPTION
**Pull request by nazarovvi**

### CODE REVIEW

**Comment by puneet0191** [comment]
@nazarovvi you have conflicts on your PR, please resolve them

**Comment by nazarovvi** [comment]
https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-ui-tests-matrix/detail/pmm2-ui-tests-matrix/474/pipeline/23

Failing tests seems to be unaffected by changes in this PR

File: codeceptConfigHelper.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ module.exports = {
     inventoryAPI: './tests/pages/api/inventoryAPI.js',
     mysqlTableDetailsPage: './tests/pages/mysqlTableDetailsPage.js',
     leftNavMenu: './tests/pages/leftNavMenu.js',
-    links: './linksHelper.js',
+    links: './tests/helper/linksHelper.js',
```
Comment (puneet0191) [inline codeceptConfigHelper.js:30]: @nazarovvi any reason why you want this to be a helper?

File: tests/helper/tooltipHelper.js [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+                slackUrl: {
+                    tabButton: locate('li').find('a').withAttr({'aria-label': 'Tab Slack'}).as('Slack Tab'),
+                    iconLocator: locate('div').after(locate('span').withText('URL')).as('Slack URL tooltip'),
+                    text: 'Slack incoming webhook URL',
+                    link: links.communicationDocs,
+                },
+            },
+        },
+    },
+}
```
Comment (puneet0191) [inline tests/helper/tooltipHelper.js:139]: empty line

File: codeceptConfigHelper.js [hunk 3]
```diff
@@ -58,6 +58,7 @@ module.exports = {
     settingsAPI: './tests/pages/api/settingsAPI.js',
     templatesAPI: './tests/ia/pages/api/templatesAPI.js',
     qanAPI: './tests/QAN/api/qanAPI.js',
+    tooltips: './tests/helper/tooltipHelper.js'
```
Comment (puneet0191) [inline codeceptConfigHelper.js:61]: the same here, why can't they just be a page object? or do you think they don't fall in the category of a page object?

**Review by puneet0191 — CHANGES_REQUESTED** [review]
minor comments

File: codeceptConfigHelper.js (hunk 1 above)
Comment (nazarovvi) [reply codeceptConfigHelper.js:30]: Tidiness mainly. All _something_Helper.js belong in the helpers folder. Easier to find

File: tests/helper/tooltipHelper.js (hunk 2 above)
Comment (nazarovvi) [reply tests/helper/tooltipHelper.js:139]: Moved tooltipHelper contents back into pageObject

File: codeceptConfigHelper.js (hunk 3 above)
Comment (nazarovvi) [reply codeceptConfigHelper.js:61]: Moved tooltipHelper contents back into pageObject

**Review by puneet0191 — APPROVED** [review]
@nazarovvi good work, have you verified the changes on the job after last push?

**Pull request merged**